contents/blog/posthog-vs-heap.md

```diff
@@ -0,0 +1,834 @@
+            <td></td>
+        </tr>
+    </thead>
+    <tbody>
+        <tr>
+            <td><strong>SOC 2 certification</strong></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td>Externally audited security</td>
+        </tr>
+        <tr>
+            <td><strong>SAML/SSO</strong></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td>Organization single sign-on</td>
+        </tr>
+        <tr>
+            <td><strong>User permissions</strong></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td>Assign access levels to team members</td>
+        </tr>
+        <tr>
+            <td><strong>Multiple organizations</strong></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td>Create orgs for all your internal teams</td>
+        </tr>
+        <tr>
+            <td><strong>Multiple projects</strong></td>
+            <td className="text-center">2 (pay for more)</td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td>Track multiple products</td>
+        </tr>
+    </tbody>
+</table>
+</div>
+
+<ArrayCTA />
+
+## Frequently asked questions
+
+### How long does it take to deploy PostHog?
+
+Deploying PostHog takes mere minutes. Just [include our tracking snippet](/docs/integrate?tab=snippet) in your product or website and (thanks to autocapture) you're immediately capturing events. Once up and running, you can enable session recording via your project settings. See our guide to [getting started on PostHog Cloud](/docs/getting-started/cloud) for more info.
+
+### How much do Heap and PostHog cost?
+
+Unlike PostHog, Heap doesn't share its pricing publicly, but its free tier is less generous.
+
+Heap's free tier allows for only 10,000 sessions per month, "limited analysis", six months of data history, and limited access to data sources and integrations.
+
+In comparison, PostHog offers 1 million events per month and 15,000 session recordings for free on all plans with no limits on data integrations – a typical medium-size B2B product with 10,000 monthly active users will generate ~500k events each month.
+
+PostHog users who input their card unlock more features and can use billing limits to avoid surprise bills.
+
+See [PostHog's pricing page](/pricing) for a full breakdown and access to our pricing calculator.
+
+### Does PostHog use cookies?
```

Comment:
Should this be its own section?

Reply:
I don't think so. While it's a nice to have, cookieless tracking is a serious USP for us.